lib/modern_searchlogic/ordering.rb

```diff
@@ -0,0 +1,20 @@
+module ModernSearchlogic
+  module Ordering
+    def self.included(base)
+      base.module_eval do
+        define_method(:order_with_modern_searchlogic) do |*args|
+          args.reduce(self) do |scope, arg|
```

Comment:
The reason this uses `reduce` is because `order(...)` technically supports variable length arguments. This allows each argument to be treated as MSL-able expression.